pkg/sql/parser/sql.y

```diff
@@ -1230,17 +1231,21 @@ preparable_stmt:
   {
     $$.val = $1.slct()
   }
-| create_stmt
-| drop_stmt
-| alter_table_stmt
 | insert_stmt
 | update_stmt
 | delete_stmt
+
+explainable_stmt:
+  preparable_stmt
+| create_stmt
+| drop_stmt
+| alter_table_stmt
```

Comment:
No known reason why this won't work, but today we don't support placeholders in DDLs.

I'd be happy to see this changed later, but I think that allowing everything to be preparable (in #16524) might have been premature - I probably would have made some noise there if I had noticed what was going on. I think we need more test coverage and thought before we enable preparing for every kind of statement.